modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/InteractListener.java

```diff
@@ -99,30 +118,24 @@ public void onPlayerInteractStick(PlayerInteractEvent event) {
                 int DY = 1;
                 if (event.getPlayer().isSneaking())
                     DY = -1;
-
+                DY *= craft.getCurrentGear();
```

Comment:
Similarly, it VERY much should not affect the distance of moving with a stick.  For example, there are bugs with certain numbers of vertical height triggering redstone torch towers on movement, and this could introduce that issue to a properly configured server.